code/game/objects/random/random.dm

```diff
@@ -1123,4 +1123,18 @@ var/list/multi_point_spawns
 				/obj/item/clothing/suit/space/void/mining/alt,
 				/obj/item/clothing/head/helmet/space/void/mining/alt
 			)
-		)
\ No newline at end of file
+		)
+
+/obj/random/landmine
+	name = "Random Landmine"
```

Comment:
Again should probably standardize land mine or landmine. I'd recommend 'land mine' with the space, but that's debatable.